source/SkiaSharp.Views.Uno/SkiaSharp.Views.Uno.WinUI.Mobile/SkiaSharp.Views.Uno.WinUI.Mobile.csproj

```diff
@@ -0,0 +1,62 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net6.0-android;net6.0-ios;net6.0-maccatalyst;net6.0-macos</TargetFrameworks>
+    <RootNamespace>SkiaSharp.Views.UWP</RootNamespace>
+    <AssemblyName>SkiaSharp.Views.UWP</AssemblyName>
+    <PackagingGroup>SkiaSharp.Views.Uno.WinUI</PackagingGroup>
+    <PackageId>SkiaSharp.Views.Uno.WinUI</PackageId>
```

Comment:
This may need a `<SignAssembly>false` to disable strong naming for Uno things. I see it is in the skia/wasm projects...